src/ADAL.PCL.Android/ADAL.PCL.Android.csproj

```diff
@@ -52,9 +52,7 @@
     <Compile Include="..\ADAL.Common\CommonAssemblyInfo.cs">
       <Link>CommonAssemblyInfo.cs</Link>
     </Compile>
-    <Compile Include="..\ADAL.Common\WebProxyProvider.cs">
-      <Link>WebProxyProvider.cs</Link>
```

Comment:
Previously the Android assembly was using a shared copy of the class. However, it needs to have an Android-specific attribute applied so I've created a copy in the Android platform project.